unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/BinaryMessage/KernelBinaryMessageDeserializer.cs

```diff
@@ -24,18 +24,18 @@ public static IEnumerator<object> Deserialize(IBinaryReader reader)
             while (reader.CanRead())
             {
                 int messageLength = reader.ReadInt32();
-                int messageType = reader.ReadInt32();
+                CrdtMessageType messageType = (CrdtMessageType)reader.ReadInt32();
```

Comment:
Will this fail if somebody (inadvertently) adds a new value to the enum?

Reply:
It shouldn't since C# won't try to convert the value to match a name on the enum.
So: 
1 ==  CrdtMessageType.PUT_COMPONENT
2 ==  CrdtMessageType.REMOVE_COMPONENT
3 == (CrdtMessageType) 3